app/routes/application.js

```diff
@@ -37,7 +37,8 @@ export default Route.extend(ApplicationRouteMixin, {
       notifications,
       pages: this.get('store').query('page', {
         sort: 'index'
-      })
+      }),
+      socialLinks: this.get('store').queryRecord('setting', {})
```

Comment:
@CosmicCoder96 Can we use `findAll` here?

Reply:
@geekyd No. A single record is expected.

Reply:
Ahhh yeah!